src/core/config/platform.h

```diff
@@ -176,4 +176,44 @@
 #endif
 #endif
 
+/**
+ * @def OPENTHREAD_PLATFORM_CONFIG_POWER_CALIBRATION_ENABLE
+ *
+ * Define to 1 to enable power calibration support.
+ *
+ */
+#ifndef OPENTHREAD_PLATFORM_CONFIG_POWER_CALIBRATION_ENABLE
+#define OPENTHREAD_PLATFORM_CONFIG_POWER_CALIBRATION_ENABLE 0
+#endif
+
+/**
+ * @def OPENTHREAD_PLATFORM_CONFIG_POWER_CALIBRATION_PLATFORM_API_ENABLE
+ *
+ * Define to 1 to enable the default implementation of the power calibration radio platform APIs.
+ *
+ */
+#ifndef OPENTHREAD_PLATFORM_CONFIG_POWER_CALIBRATION_PLATFORM_API_ENABLE
+#define OPENTHREAD_PLATFORM_CONFIG_POWER_CALIBRATION_PLATFORM_API_ENABLE 0
+#endif
```

Comment:
How about defining `OPENTHREAD_CONFIG_POWER_CALIBRATION_ENABLE` in a new file `confg/power_calibration.h` (which enables/disables power calib feature in OT itself under `Util`). And then have `NUM_POWER_ENTRIES` and SETTING_SIZE` as its sub-paramters (I think they are only used by the `Utls` module, right?)